chapter_introduction/index_vn.md

```diff
@@ -2274,8 +2274,8 @@ over the past decade.
   throughout the network, replacing weights by random variables
   for training purposes.
 -->
-
-*dịch đoạn phía trên*
+* Các phương pháp tiên tiến trong việc kiểm soát sức chứa, như Dropout :cite:`Srivastava.Hinton.Krizhevsky.ea.2014`, đã giúp làm giảm sự nguy hiểm của quá khớp.
```

Comment:
```suggestion
* Các phương pháp tiên tiến trong việc kiểm soát dung lượng, như Dropout :cite:`Srivastava.Hinton.Krizhevsky.ea.2014`, đã giúp làm giảm sự nguy hiểm của quá khớp.
```
Từ Dung lượng có hay hơn là sức chứa không nhỉ?

Reply:
`sức chứa` hơi mơ hồ, `dung lượng` thì liên tưởng tới bộ nhớ bytes, `năng lực` thì sao nhỉ?
```suggestion
* Các phương pháp tiên tiến trong việc kiểm soát năng lực, như Dropout :cite:`Srivastava.Hinton.Krizhevsky.ea.2014`, đã giúp làm giảm sự nguy hiểm của quá khớp.
```

Reply:
`Năng lực` thì thật sự mơ hồ....

Reply:
Mình nghĩ từ "năng lực" ok. Mỗi mô hình có một mức năng lực nhất định, muốn tăng năng lực thì phải làm cho mô hình to hơn. Mình sẽ còn năng lực và move on.